app/home/index.html

```diff
@@ -98,9 +98,9 @@
   <script src="../common/domain/services/encounterService.js"></script>
   <script src="../common/domain/services/consultationService.js"></script>
 
-
-  <script src="../common/config/init.js"></script>
-  <script src="../common/config/services/configurations.js"></script>
+  <script src="../common/patient/constants.js"></script>
+  <script src="../common/patient/models/patientConfig.js"></script>
+  <script src="../common/patient/mappers/patientAttributeTypeMapper.js"></script>
```

Comment:
Are you alphabetizing these files?

Reply:
Not alphabetizing, I think it's a coincidence.